airtime_mvc/application/models/Preference.php

```diff
@@ -498,7 +498,12 @@ public static function SetStationDescription($description)
 
     public static function GetStationDescription()
     {
-        return self::getValue("description");
+        $description = self::getValue("description");
+        if (!empty($description)) {
+            return $description;
+        } else {
+            return "Powered by Airtime Pro";
```

Comment:
This is what sprintf is for. You put the % placeholder in the translatable string and that allows the translator to place the "Airtime Pro" string correctly according to their grammar.